### PR TITLE
Add shane32 to 2023-10-26 meeting

### PR DESCRIPTION
**Pull request by Shane32**
@benjie I listed GraphQL.NET as my organization, as a maintainer of those repositories.  However, if you'd rather that I list the company I work for, I can do that instead.

### CODE REVIEW

**Comment by benjie** [comment]
Please either put company or company/project. I think the affiliation is for legalish stuff.

File: working-group/agendas/2023/2023-10-26.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ Benjie Gillam           | @Benjie                | Graphile                | Sou
 Jovi De Croock          | @JoviDeCroock          | Stellate                | Aalst, BE
 Denis Badurina          | @enisdenjo             | The Guild               | Sarajevo, BA
 Gabriel McAdams         | @ghmcadams             | self                    | San Jose, CA, USA
+Shane Krueger           | @shane32               | GraphQL.NET             | Detroit, MI, USA
```
Comment (Shane32) [inline working-group/agendas/2023/2023-10-26.md:104]:
```suggestion
Shane Krueger           | @shane32               | Zbox Global LLC                        | Detroit, MI, USA
```

File: working-group/agendas/2023/2023-10-26.md [hunk 2]
```diff
@@ -101,6 +101,7 @@ Benjie Gillam           | @Benjie                | Graphile                | Sou
 Jovi De Croock          | @JoviDeCroock          | Stellate                | Aalst, BE
 Denis Badurina          | @enisdenjo             | The Guild               | Sarajevo, BA
 Gabriel McAdams         | @ghmcadams             | self                    | San Jose, CA, USA
+Shane Krueger           | @shane32               | Zbox Global LLC                        | Detroit, MI, USA
```
Comment (Shane32) [inline working-group/agendas/2023/2023-10-26.md:104]:
```suggestion
Shane Krueger           | @shane32               | Zbox Global LLC           | Detroit, MI, USA
```

File: working-group/agendas/2023/2023-10-26.md [hunk 3]
```diff
@@ -101,6 +101,7 @@ Benjie Gillam           | @Benjie                | Graphile                | Sou
 Jovi De Croock          | @JoviDeCroock          | Stellate                | Aalst, BE
 Denis Badurina          | @enisdenjo             | The Guild               | Sarajevo, BA
 Gabriel McAdams         | @ghmcadams             | self                    | San Jose, CA, USA
+Shane Krueger           | @shane32               | Zbox Global LLC           | Detroit, MI, USA
```
Comment (Shane32) [inline working-group/agendas/2023/2023-10-26.md:104]:
```suggestion
Shane Krueger           | @shane32               | Zbox Global LLC         | Detroit, MI, USA
```

**Comment by Shane32** [comment]
Just found out I can't make it :-(

**Comment by benjie** [comment]
Hope you can make it next time! 🤞

**Pull request closed without merging**